src/main/java/com/uber/cadence/samples/hello/HelloSideEffect.java

```diff
@@ -0,0 +1,73 @@
+package com.uber.cadence.samples.hello;
+
+import static com.uber.cadence.samples.common.SampleConstants.DOMAIN;
+
+import com.uber.cadence.client.WorkflowClient;
+import com.uber.cadence.worker.Worker;
+import com.uber.cadence.workflow.QueryMethod;
+import com.uber.cadence.workflow.Workflow;
+import com.uber.cadence.workflow.WorkflowMethod;
+
+/**
+ * Hello SideEffect Cadence workflow that sets a SideEffect. The set value can be queried Requires a
+ * local instance the Cadence service to be running.
+ */
+public class HelloSideEffect {
+
+  static final String TASK_LIST = "HelloSideEffect";
+
+  /** Workflow interface has to have at least one method annotated with @WorkflowMethod. */
+  public interface SideEffectWorkflow {
+    @WorkflowMethod(executionStartToCloseTimeoutSeconds = 10, taskList = TASK_LIST)
+    void set(String value);
+
+    /** @return set value */
+    @QueryMethod
+    String get();
+  }
+
+  public static class SideEffectWorkflowImpl implements SideEffectWorkflow {
+
+    private String value = "";
+
+    @Override
+    public void set(String value) {
+      Workflow.sideEffect(
+          String.class,
+          () -> {
+            return value;
```

Comment:
Can you change it to generate a UUID? That might make more sense.
I don't understand why user want to pass the value into side effect. By definition that value cannot change after workflow start.

Reply:
Sure, did not really think about that. It was just for testing the UI.